src/sys/ptrace.rs

```diff
@@ -134,8 +134,8 @@ pub unsafe fn ptrace(request: Request, pid: Pid, addr: *mut c_void, data: *mut c
     }
 }
 
-fn ptrace_peek(request: Request, pid: Pid, addr: *mut c_void, data: *mut c_void) -> Result<c_long> {
-    let ret = unsafe {
+unsafe fn ptrace_peek(request: Request, pid: Pid, addr: *mut c_void, data: *mut c_void) -> Result<c_long> {
```

Comment:
Please separate this into a separate commit and also add a separate changelog entry for it under CHANGED.

Reply:
Indeed you're correct assuming you meant it *is* a part of our private API.

Reply:
What's the point of announcing our implementation details?